src/config_trapdoor.c

```diff
@@ -268,6 +270,7 @@ TbBool parse_trapdoor_trap_blocks(char *buf, long len, const char *config_textna
           trapst->bigsym_sprite_idx = 0;
           trapst->medsym_sprite_idx = 0;
           trapst->pointer_sprite_idx = 0;
+          trapst->place_sound_idx = 0;
```

Comment:
maybe the default should be 117, so this PR doesn't change custom traps bundled with maps

Reply:
I'll change that!